node/web/src/lib.rs

```diff
@@ -119,18 +127,34 @@ async fn setup_node(
         .work_verifier_index(work_verifier_index.clone());
 
     // TODO(binier): refactor
-    if let Some(seed_nodes_url) = seed_nodes_url {
-        let peers = ::node::core::http::get_bytes(&seed_nodes_url)
-            .await
-            .expect("failed to fetch seed nodes");
-        node_builder.initial_peers(
-            String::from_utf8_lossy(&peers)
-                .split("\n")
-                .filter(|s| !s.trim().is_empty())
-                .map(|s| s.trim().parse().expect("failed to parse seed node addr")),
-        );
+    let mut all_raw_peers = seed_nodes_fixed.unwrap_or_default();
+    if let Some(seed_nodes_urls) = seed_nodes_urls {
+        for seed_nodes_url in seed_nodes_urls {
+            let peers = ::node::core::http::get_bytes(&seed_nodes_url).await;
```

Comment:
another nit
```suggestion
            let peers = node::core::http::get_bytes(&seed_nodes_url).await;

```
I've really only seen the preceeding `::` in macro code before. We don't need it, right?

Reply:
We do. There's some overloading of the "node" label going on in this file, where you have `node`-the-crate (i.e. `node/Cargo.toml`, `::node`) and `node`-the-mod-in-web-lib (`node/web/src/node/mod.rs`, aka `mina_core_web::node` / `crate::node`). I did not want to mess with refactoring that for clarity at the time.